### PR TITLE
Removed the double quotes

### PR DESCRIPTION
**Pull request by egyedia**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/egyedia/egyedia.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: Attila L. Egyedi
+institution: Georgia Tech
+profile_pic: atti.png
+quote: Give me six hours to chop down a tree and I will spend the first four sharpening the ax. – Lincoln
```
Comment (Mohammed-Shoaib) [inline _data/egyedia/egyedia.md:5]: Change em dash – to single dash - as it is a non-ascii character.

Comment (egyedia) [reply _data/egyedia/egyedia.md]: Thank you for the detailed error report, I fixed it!

**Comment by egyedia** [comment]
Thank you for the review!
Do I need to do something more, or this will be approved and merged automatically?

Regards,
Attila

**Comment by Mohammed-Shoaib** [comment]
@egyedia You don't have to do anything else, your pull request will be merged by one of the maintainers later! :confetti_ball::+1:

Till then sit back, relax, and enjoy :relaxed:

File: _data/egyedia/egyedia.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+---
+name: Attila L. Egyedi
+institution: Georgia Tech
+profile_pic: atti.png
+quote: Give me six hours to chop down a tree and I will spend the first four sharpening the ax. - Lincoln
```
Comment (juanpflores) [inline _data/egyedia/egyedia.md:5]:
```suggestion
quote: Give me six hours to chop down a tree and I will spend the first four sharpening the axe. - Lincoln
```

**Pull request merged**